src/iris/role_lookup/oncall.py

```diff
@@ -80,6 +80,12 @@ def get(self, role, target):
                 return [user['user'] for user in result]
             else:
                 return None
+        elif role == 'director':
+            result = self.call_oncall('/teams/%s/oncall/director' % target, list)
+            if result:
+                return [user['user'] for user in result]
+            else:
+                return None
```

Comment:
```suggestion
        elif role in ['director', 'manager']:
            result = self.call_oncall('/teams/%s/oncall/%s' % (target, role), list)
            if result:
                return [user['user'] for user in result]
            else:
                return None
```

Or something like it might bit a bit cleaner.

Reply:
changed